fedn/cli/run_cmd.py

```diff
@@ -132,16 +135,17 @@ def startup_cmd(ctx, path):
     dispatcher = Dispatcher(config, path)
     _ = dispatcher._get_or_create_python_env()
     dispatcher.run_cmd("startup")
-
-    # delete the virtualenv
-    if dispatcher.python_env_path:
-        logger.info(f"Removing virtualenv {dispatcher.python_env_path}")
-        shutil.rmtree(dispatcher.python_env_path)
+    if venv:
+        # delete the virtualenv
+        if dispatcher.python_env_path:
+            logger.info(f"Removing virtualenv {dispatcher.python_env_path}")
+            shutil.rmtree(dispatcher.python_env_path)
 
 @run_cmd.command("build")
 @click.option("-p", "--path", required=True, help="Path to package directory containing fedn.yaml")
+@click.option("-v", "--venv", default=True,type=bool,required=False, help="flag if set to False doesn't remove venv")
```

Comment:
add space after ",", are you not using ruff in your IDE?

Reply:
yes I am using ruff in IDE